conjure-codegen/src/example_types/another/test_service.rs

```diff
@@ -1192,2212 +1201,746 @@ pub trait TestService<I, O> {
     ) -> Result<Option<Self::MaybeGetRawDataBody>, conjure_http::private::Error>;
+    #[endpoint(
+        method = GET,
+        path = "/catalog/datasets/{datasetRid}/string-aliased",
+        name = "getAliasedString",
+        produces = conjure_http::server::StdResponseSerializer
+    )]
     fn get_aliased_string(
         &self,
+        #[auth]
         auth_: conjure_object::BearerToken,
+        #[path(
+            name = "datasetRid",
+            decoder = conjure_http::server::conjure::FromPlainDecoder,
+            log_as = "datasetRid"
+        )]
         dataset_rid: conjure_object::ResourceIdentifier,
     ) -> Result<super::super::product::AliasedString, conjure_http::private::Error>;
+    #[endpoint(
+        method = POST,
+        path = "/catalog/datasets/upload-raw",
+        name = "uploadRawData"
+    )]
     fn upload_raw_data(
         &self,
+        #[auth]
         auth_: conjure_object::BearerToken,
+        #[body(deserializer = conjure_http::server::conjure::BinaryRequestDeserializer)]
         input: I,
     ) -> Result<(), conjure_http::private::Error>;
+    #[endpoint(
+        method = POST,
+        path = "/catalog/datasets/upload-raw-aliased",
+        name = "uploadAliasedRawData"
+    )]
     fn upload_aliased_raw_data(
         &self,
+        #[auth]
         auth_: conjure_object::BearerToken,
+        #[body(deserializer = conjure_http::server::conjure::BinaryRequestDeserializer)]
         input: I,
     ) -> Result<(), conjure_http::private::Error>;
+    #[endpoint(
+        method = GET,
+        path = "/catalog/datasets/{datasetRid}/branches",
+        name = "getBranches",
+        produces = conjure_http::server::conjure::CollectionResponseSerializer
+    )]
     fn get_branches(
         &self,
+        #[auth]
         auth_: conjure_object::BearerToken,
+        #[path(
+            name = "datasetRid",
+            decoder = conjure_http::server::conjure::FromPlainDecoder,
+            log_as = "datasetRid"
+        )]
         dataset_rid: conjure_object::ResourceIdentifier,
     ) -> Result<std::collections::BTreeSet<String>, conjure_http::private::Error>;
     ///Gets all branches of this dataset.
+    #[endpoint(
+        method = GET,
+        path = "/catalog/datasets/{datasetRid}/branchesDeprecated",
+        name = "getBranchesDeprecated",
+        produces = conjure_http::server::conjure::CollectionResponseSerializer
+    )]
     fn get_branches_deprecated(
         &self,
+        #[auth]
         auth_: conjure_object::BearerToken,
+        #[path(
+            name = "datasetRid",
+            decoder = conjure_http::server::conjure::FromPlainDecoder,
+            log_as = "datasetRid"
+        )]
         dataset_rid: conjure_object::ResourceIdentifier,
     ) -> Result<std::collections::BTreeSet<String>, conjure_http::private::Error>;
+    #[endpoint(
+        method = GET,
+        path = "/catalog/datasets/{datasetRid}/branches/{branch:.+}/resolve",
```

Comment:
What does the `{branch:.+}` do?

Reply:
It allows the param to match multiple path segments (e.g. `/branches/foo/bar/resolve`, `branch` would be `foo/bar`). It's not really used in modern conjure, and IIRC most implementations won't support it but I haven't updated the example definitions in a long time.